CIP-0000.md

```diff
@@ -0,0 +1,198 @@
+- - - -
+lang: en
+cip: 0000
+title: Celo Improvement Proposal (CIP) Process
+status: Draft
+type: Meta
+discussions-to:
+author: Yaz Khoury (@YazzyYaz), James Prestwich (@prestwich), Pranay Mohan (@pranaymohan), Mariano Cortesi (@mcortesi)
+created: 2020-09-22
+license: CC0 1.0
+- - - -
+
+# Abstract
+A Celo Improvement Proposal (CIP) describe standards for the Celo platform, including the core protocol specifications, SDK, and contract standards. It is a design document providing information to the Celo community, or describing a new feature for Celo or its processes or environment, a rationale for the proposal, detailed solution including technical specifications, and, if any, a list of potential risks. 
+
+The CIP proposer, the **author** is responsible for building consensus within the community and documenting whether there is rough consensus on their proposal. 
```

Comment:
We should document what "rough consensus" means, and some examples of how to document it.

Reply:
It's linked in the document.